lib/plugins/aws/provider.js

```diff
@@ -1698,6 +1744,170 @@ class AwsProvider {
       return this.getStackResources(res.NextToken, allResources);
     });
   }
+
+  getEcrRepositoryName() {
```

Comment:
We have a dedicated [naming.js](https://github.com/serverless/serverless/blob/bcbbd47fa09b7d99d7f8da3f11150215d1203bba/lib/plugins/aws/lib/naming.js) module for such methods